tests/ReportbackApiTest.php

```diff
@@ -22,6 +23,18 @@ class ReportbackApiTest extends TestCase
      */
     public function testCreatingNewReportback()
     {
+        // Mock sending image to AWS.
+        $this->fileSystem->shouldReceive('put')->andReturn(true);
+
+        // Create an uploaded file.
+        $path = storage_path('images/huskycorgi.jpeg');
+        $original_name = 'huskycorgi.jpeg';
+        $mime_type = 'image/jpeg';
+        $size = 2476;
+        $error = null;
+        $test = true;
+        $file = new \Illuminate\Http\UploadedFile($path, $original_name, $mime_type, $size, $error, $test);
```

Comment:
👍! It could be nice to make a `$this->mockFile` method on the TestCase so this can be re-used elsewhere.